src/Common/tests/XunitTraitsDiscoverers/Attributes/PlatformSpecificAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Xunit.Sdk;
+
+namespace Xunit
+{
+    /// <summary>
+    /// Apply this attribute to your test method to specify this is a platform specific test.
+    /// </summary>
+    [TraitDiscoverer("Xunit.TraitDiscoverers.PlatformSpecificDiscoverer", "XunitTraitsDiscoverers")]
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
+    public class PlatformSpecificAttribute : Attribute, ITraitAttribute
+    {
+        public PlatformSpecificAttribute(PlatformID platform) { }
```

Comment:
I'll be doing that in a separate PR. This attribute says runs this test only on these platforms, the activeissue skipping will be dont run this test only on these platforms. I'll either be exposing OSVersion on cloud platform, or let the xunit options on commandline during build control it. My final syntax for ActiveIssue will be ActiveIssue((string | int)issue, PlatformID platforms) since platformID is a flags enum, a params is not required.